CONTRIBUTING.md

```diff
@@ -1,10 +1,22 @@
 # Contributing
 
+## Introduction
+
+This book is a catalogue of Rust programming techniques, (anti-)patterns, and idioms and other explanations.
+It is a compilation of collective (sometimes implicit) knowledge or experience that was created with collaborative effort.
+
+These contained information are __no rules__, they should be understood as guidelines.
+We are collecting these information so people can form a balanced view on certain aspects.
+
+If you want to be part of that effort this is how you can participate:
+
+
 ## Discussion board
 
 If you have a question or an idea regarding certain content but you want to have feedback of fellow community members 
 and you think it may not be appropriate to file an issue open a discussion in our [discussion board](https://github.com/rust-unofficial/patterns/discussions).
 
+
```

Comment:
Why is the extra spaces necessary?

Reply:
Two new lines between a headline and the section before I thought

Reply:
This should probably be enforced by a markdown formatter/linter.

Reply:
Yeah, we're working on it ;-)